events/smart-descriptions.json

```diff
@@ -1,18 +1,56 @@
 {
     "vectra cognito detect": [
-	{
-	    "value": "{observer.ip} detected {vectra.detection.name} : {host.name} ({host.ip})",
-	    "conditions": [{
-		"field": "vectra.detection.name"
-	    }]
-	},
-	{
-	    "value": "{observer.ip} refreshed detection {vectra.detection.last_type} : {host.name} ({host.ip})",
-	    "conditions": [{
-		"field": "vectra.detection.last_type"
-	    }]
-	}	
-    ],
+    {
+        "value": "{observer.ip} detected {vectra.detection.name} : {host.name} ({host.ip})",
+        "conditions": [{
+      "field": "vectra.detection.name"
+        }],
+      "relationships": [{
+          "source": "host.ip",
+          "target": "destination.ip",
+          "type": "{vectra.detection.name}"
+        }]
+    },
+    {
+        "value": "{observer.ip} refreshed detection {vectra.detection.last_type} : {host.name} ({host.ip})",
+        "conditions": [{
+      "field": "vectra.detection.last_type"
+        }]
+    },
+    {
+      "value": "[HOST SCORING] {host.name} ({host.ip}) : threat = {vectra.risk_score_norm}",
+      "conditions": [{
+      "field": "event.type",
+      "value": "HOST SCORING"
+        }],
+    },
+    {
+      "value": "[LOCKDOWN] {user.name} {action.name} {vectra.account.name} ",
```

Comment:
Smart Descriptions are supposed to abstract complexity as much as possible. Can't we create a sentence here ?